client/src/components/App/Content/SignUp/SignUp.tsx

```diff
@@ -42,9 +44,10 @@ const SignUp : React.FC<Props> = ({ open, handleSaveUser, handleCloseSignUp }) =
             </DialogTitle>
 
             <DialogContent>
-                <SignUpForm 
+                <UserPropertiesForm 
```

Comment:
If you have decided to change the component name (in this case SignUpForm) make sure you change other component's name such as SignUpFormStyles and SignUpSchema